hibernate-core/src/main/java/org/hibernate/query/sqm/tree/jpa/AbstractJpaTupleElement.java

```diff
@@ -47,7 +47,7 @@ protected void setAlias(String alias) {
 		return expressableType;
 	}
 
-	protected final void setExpressableType(SqmExpressable<?> expressableType) {
+	public final void setExpressableType(SqmExpressable<?> expressableType) {
```

Comment:
This change is a bit concerning. Why is it necessary to set the expressable type to null for `JpaCriteriaParameter`? Wouldn't it be better to have some sort of special method on `JpaCriteriaParameter` to take care of this?